### ISSUE TITLE
Newcomer Chat needs to be added to the channel names for custom names

### ISSUE THREAD

**Issue by wheatbread** [funkydude/BasicChatMods#35]
Hello!

Just wanted to let you know there's a new chat that appears in the game if someone is either a new player or signs up to be a mentor in the new mentor guide system.  The chat is called Newcomer Chat.  It would be great if we could have it in the dropdown list of default channels to make custom names for.

**Comment by funkydude** [comment]
Do you mean it's a custom chat channel like trade/general?
How does one enter this channel to test this?

**Comment by wheatbread** [comment]
Yes, it's an ingame channel that appears if you sign up to be a mentor or a completely new player.  There are some achievements required in order to be a mentor.   It's character-based (as opposed to account-wide) and you sign up at:  

`Alliance players can talk to Celeste Periwinkle by the Stormwind Embassy and Horde players can talk to Eyla Pathleader by the Orgrimmar Embassy.`

More info:  https://www.wowhead.com/news=318521/guide-mentor-system-help-new-players-starting-out-in-shadowlands
